src/components/Profile/components/PersonalInfoList/components/UpdatePlannedGraduationYear/index.jsx

```diff
@@ -57,9 +57,16 @@ const UpdatePlannedGraduationYear = (props) => {
           />
         </FormControl>
         <p className={styles.note}>
-          NOTE: <br /> This does not set your official graduation <br />
-          year. To make an official change, please <br /> contact{' '}
-          <a href="mailto:registrar@gordon.edu">the Registrar's Office</a>.
+          NOTE: <br /> This does not replace the Application to
+          <br />
+          Graduate.
+          <br />
+          Please fill that out 8-12 months before
+          <br />
+          you plan to graduate. The application <br />
```

Comment:
The "please" feels like it changes the tone from an important notification to a request. Maybe this is a clearer tone?
```suggestion
          NOTE: <br /> This does not replace the Application to
          <br />
          Graduate, which must be filled out 8-12 months before
          <br />
          you plan to graduate. The application <br />
```